irohad/ametsuchi/impl/mutable_storage_impl.cpp

```diff
@@ -130,6 +130,7 @@ namespace iroha {
           log_->warn("Apply has been failed. Reason: {}", e.what());
         }
       }
+      block_storage_->clear();
```

Comment:
Why do we have to do it manually? Seems like should be done at `block_storage_` destructor, because the resources being cleared here are exclusively possessed and managed by `block_storage_`.